sample-compose/src/main/java/com/airbnb/lottie/sample/compose/lottiefiles/LottieFilesPage.kt

```diff
@@ -74,8 +75,11 @@ fun LottieFilesTabBarTab(
 ) {
     val textWidth = remember { mutableStateOf(0) }
     val pxRatio = with(AmbientDensity.current) { 1.dp.toPx() }
-    val tabWidth by animateAsState(if (isSelected) (textWidth.value / pxRatio).dp else 0.dp, spring(), null)
-    val tabAlpha by animateAsState(if (isSelected) 1f else 0f)
+    val tabWidth by animateDpAsState(
+        if (isSelected) (textWidth.value / pxRatio).dp else 0.dp,
+        animationSpec = spring()
```

Comment:
What's wrong with the [default animation](https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/animation/animation-core/src/commonMain/kotlin/androidx/compose/animation/core/AnimateAsState.kt;l=114?q=animateDpAsState)?

Reply:
Default animation is cool for me, but the `tabWidth` animation was using `spring()` before!

Reply:
Let's just stick with the default for simplicity!

Reply:
Sure, sounds good!